bootstrap/config/overlays/gcp/gcp_prototype.yaml

```diff
@@ -0,0 +1,59 @@
+# Sample config for kubeflow bootstrapper for v0.4
```

Comment:
do we have a type definition like this?
https://github.com/kubeflow/kubeflow/blob/3d56531ab8a35dc2663a6e764284051ed020ba34/bootstrap/config/types.go#L28

If we are gonna parse it in apps such as kfctl or bootstraper, could we define types as well?